worlds/oot/__init__.py

```diff
@@ -842,7 +841,7 @@ def generate_basic(self):  # mostly killing locations that shouldn't exist by se
         all_state.sweep_for_events(locations=all_locations)
         reachable = self.multiworld.get_reachable_locations(all_state, self.player)
         unreachable = [loc for loc in all_locations if
-                       (loc.internal or loc.type == 'Drop') and loc.event and loc.locked and loc not in reachable]
+                       (loc.internal or loc.type == 'Drop') and loc.advancement and loc.locked and loc not in reachable]
```

Comment:
Similar to stardew, while this may have the same effect, I think the naming is now all wrong. (If i read the comment a few lines up correctly).